core/src/trezor/ui/layouts/tt_v2/webauthn.py

```diff
@@ -13,8 +19,39 @@ async def confirm_webauthn(
     info: ConfirmInfo,
     pageable: Pageable | None = None,
 ) -> bool:
-    raise NotImplementedError
+    if pageable is not None:
+        # TODO: how to even replicate this situation?
+        # Called from Fido2ConfirmGetAssertion and Fido2ConfirmNoCredentials
+        # cbor_get_assertion_process -> cbor_get_assertion ->
+        # dispatch_cmd -> req.data[0] == _CBOR_GET_ASSERTION
+        # TODO: should create a UI test if relevant
+        raise NotImplementedError
+
+    assert info.app_icon is not None
+    confirm = _RustLayout(
+        trezorui2.confirm_webauthn(
+            title=info.get_header(),
+            app_name=info.app_name(),
+            account_name=info.account_name(),
+            icon=info.app_icon,
+        )
+    )
+
+    if ctx is None:
+        return is_confirmed(await confirm)
+    else:
+        return is_confirmed(
+            await interact(ctx, confirm, "confirm_webauthn", ButtonRequestType.Other)
+        )
 
 
 async def confirm_webauthn_reset() -> bool:
-    raise NotImplementedError
+    confirm = _RustLayout(
+        trezorui2.confirm_action(
+            title="FIDO2 Reset",
```

Comment:
In the new design all titles are uppercase.

Reply:
Changed in [3d79a2d](https://github.com/trezor/trezor-firmware/pull/2556/commits/3d79a2d30c9e4a6a8290fd9a32e1bfcf1355476f)